docs/configuration.md

```diff
@@ -120,6 +120,29 @@ jobs:
   trigger: commit
 ```
 
+**copr\_build**
```

Comment:
to be honest, I'd love to do `build` here; the fact that it's copr should be our implementation detail

but this can be changed in a next PR, I'd just love to express my intent here, so you're not surprised on monday

Reply:
Oh, I was confused with already existing keywords for build in the code. I thought you meant something else by it :D